components/workouts/ExerciseDetails.jsx

```diff
@@ -0,0 +1,78 @@
+/* eslint-disable react/forbid-prop-types, react/no-array-index-key */
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components/native';
+import SetDetails from './SetDetails';
+import LogHeader from './LogHeader';
+import PlusButton from '../utils/PlusButton';
+
+const ExerciseName = styled.Text`
+  font-family: 'Montserrat_500Medium'
+  font-size: 26px;
+  color: #FFF;
+  margin: 15px;
+`;
+
+const Container = styled.View`
+  border-radius: 20px;
+  margin: 10px 3%;
+  box-shadow: 3px 5px 2px #00000050;
+`;
+
+const ButtonFlexContainer = styled.View`
+  align-items: flex-end;
+  margin: 10px;
+`;
+
+const AddSetButton = styled(PlusButton)`
+  padding: 5px 10px;
+`;
+
+const ExerciseDetails = (props) => {
+  const {
+    items, updateReps, updateWeight, updateCompleted, onSetAdd, onSetDelete, name, color,
+  } = props;
+
+  return (
+    <Container style={{ backgroundColor: color }}>
+      <ExerciseName>{name}</ExerciseName>
+      <LogHeader />
+      {items.map((set, index) => (
+        <SetDetails
+          prevWeight={set.prevWeight}
+          prevReps={set.prevReps}
+          weight={set.weight}
+          reps={set.reps}
```

Comment:
Did we decide on a way to log time for timed workouts? We might want that in SetDetails

Reply:
Oh yeah I forgot about that, I'll update this pr with that